root/defaults/nginx/ssl.conf.sample

```diff
@@ -1,11 +1,8 @@
-## Version 2021/11/25 - Changelog: https://github.com/linuxserver/docker-baseimage-alpine-nginx/commits/master/root/defaults/nginx/server-confs/ssl.conf.sample
+## Version 2022/08/05 - Changelog: https://github.com/linuxserver/docker-baseimage-alpine-nginx/commits/master/root/defaults/ssl.conf.sample
 
 ### Mozilla Recommendations
-# generated 2021-10-16, Mozilla Guideline v5.6, nginx 1.20.1-r3, OpenSSL 1.1.1l-r0, intermediate configuration
-# https://ssl-config.mozilla.org/#server=nginx&version=1.20.1-r3&config=intermediate&openssl=1.1.1l-r0&guideline=5.6
-
-listen 443 ssl http2;
-listen [::]:443 ssl http2;
```

Comment:
This would have made any conf that includes `ssl.conf` listen on 443, which was in-line with the idea of including `ssl.conf` if you want https, but the method of including files resulted in no clear way to make inclusion optional on an individual conf level (ex: include a conf for one proxy but not another).